params/config.go

```diff
@@ -93,42 +92,76 @@ var (
-				Enode:         "enode://74a4f767ad2f3f607a2db06732b44e6c61a68cae1959b331c18aea6256aae16bded31ba40dd85dcc4d719baaeb29f918726d19fa51b5d8174b27da0d7593e19b@34.142.33.89:30303",
-				BondedStake:   bigBondedStake,
-			}, {
-				Treasury:      common.HexToAddress("0x59e2EE43e1950a348ab3CC9b6Eb847C019c2CFb1"),
-				OracleAddress: common.HexToAddress("0x7615a195832843798DDb75B11CDA87D61C5F794D"),
-				Enode:         "enode://0ddc30943837f9416f563063ed5d409aca37780b8b8f939ef9f4b7901b9eb94c09d7ba2af27f70b33d76e74403d00021c13ebc4943ad46bc1e5051689cd862b8@35.234.131.29:30303",
-				BondedStake:   bigBondedStake,
-			}, {
-				Treasury:      common.HexToAddress("0xEd1c2E7143Ad134909a39AadC25599E36064803E"),
-				OracleAddress: common.HexToAddress("0x3c44B519B6b52EF347DaD0478E80455DbBf037f9"),
-				Enode:         "enode://9435658d26e5daf30261648504560f6375b24cdf0e4403613d44ebc4020489cc67ac82ababe7928d63d9f113c67b946845d18db935abe3d241e665114fc75e94@35.177.73.222:30303",
-				BondedStake:   bigBondedStake,
-			}, {
-				Treasury:      common.HexToAddress("0xb34ed221c46a6Ad530d250856f94baD84E4308d5"),
-				OracleAddress: common.HexToAddress("0x7732255C36e5796806c19195503D3a323Fc5c616"),
-				Enode:         "enode://fe2c621f2b660725a3d529b3eefd780e90bb86e9eb4b7136c0b00a7365260a478b9b8941f1a65c6d4d77bff1b2e22eb6d781f5cc86401d60b373c6d4155c189a@3.10.195.56:30304",
-				BondedStake:   bigBondedStake,
-			}, {
-				Treasury:      common.HexToAddress("0xb94475118860Aa51544AF1C16eE3Db3799B6Db64"),
-				OracleAddress: common.HexToAddress("0x256702DdF28F9282E582214Be52aE15Bf1386a25"),
-				Enode:         "enode://6ab1e6bbf5897e1a24ccf8d8718615ec972ffd54d99c3e46f4517d5602e8bf7110e2e5e2c2e584795e45e2e842172de044b4df165a7082133c6697b632da8282@18.168.88.205:30305",
-				BondedStake:   bigBondedStake,
-			}},
+			Validators: []*Validator{
+				{
+					// CTL GV-0
+					Treasury:      common.HexToAddress("0x75474aC55768fAb6fE092191eea8016b955072F5"),
+					OracleAddress: common.HexToAddress("0x6c5AE53a803796D788E917D1fE919BfC8B56d2E6"),
+					ConsensusKey:  common.Hex2Bytes("a3aa75e42e99275f7d7985538fedc06e7f128b138a5311702afc0dc129484763645c40c36fdd97ff0d0293b00a031714"),
+					Enode:         "enode://772248dfe1af5f77e0efc0510e83364bfad55cbd6d3e276f3bd0b4ddec6472aa98645655fd80bbf049ba3da18d219ab30a68fcb98da8e06dd42863dd0356cc95@35.242.168.170:30303",
+					BondedStake:   Ntn10000,
+				}, {
+					// CTL GV-1
+					Treasury:      common.HexToAddress("0x821BC352E77D885906B47001863f75e15C114f70"),
+					OracleAddress: common.HexToAddress("0x7C056299014D2F6f2e506ef1A4F89c94AAca004e"),
+					ConsensusKey:  common.Hex2Bytes("8a7474c5d53279bd21b8e0d0475ca6cd868155ac16d67d22c15eaee75f87101d9b329f4c2e2da52934a845fc4567254a"),
+					Enode:         "enode://22f696529d7874ca66d177c2c272600c3d1f2f7111d02140c462a8cbc789f5f8968c2ce57a5aac1373ef17bf3fc67d155b54691d1413516459824067e13750a4@34.92.27.46:30303",
+					BondedStake:   Ntn10000,
```

Comment:
```suggestion
					BondedStake:   Ntn1,
```

Changing from `Ntn10000` to `NTN1` as requested by @spenot09